customer_data_analyze.py

```diff
@@ -0,0 +1,85 @@
+import pandas as pd
+
+path_to_folder = "/Users/markofir/py/"
+file_name = "customer_data.csv"
+
+df = pd.read_csv(path_to_folder + file_name)
+
+## Biggest Purchase Ammount
+
+biggest_so_far = 0
+for index, row in df.iterrows():
+    purchas_amount = row["purchase_amount"]
+    if purchas_amount > biggest_so_far:
+        biggest_so_far = purchas_amount
+        print(str(index) + " " + str(biggest_so_far))
+    
+## Average Age Of All Customers
+
+sum_of_ages = 0
+number_of_columns = 100000
```

Comment:
number of **rows**
אבל למה בכלל צריך לשמור במשתנה? אפשר להשתמש ב len או לייצר counter
הרי אם הטבלה תשתנה - תצטרך לעדכן את הקוד ואת המספר הזה. לא טוב